InvenTree/templates/InvenTree/settings/device_barcode_scanner.html

```diff
@@ -0,0 +1,24 @@
+{% extends "panel.html" %}
+
+{% load i18n %}
+{% load inventree_extras %}
+
+{% block label %}device-barcode-scanner{% endblock %}
+
+{% block heading %}
+{% trans "Barcode Scanner" %}
+{% endblock %}
+
+{% block content %}
+
+<div class='row'>
+    <table class='table table-striped table-condensed'>
+        <tbody>
+            {% include "InvenTree/settings/setting.html" with key="BARCODE_DETECTION_ENABLED" icon='fa-tag' local=True %}
```

Comment:
I think that adding "local" settings is going to complicate things a *lot* - very hard to debug too!

Unless the extra overhead can be justified, I'd rather see these be settings stored on the server